src/components/Player/Pages/Wardmap/Wardmap.jsx

```diff
@@ -14,6 +15,7 @@ const getData = (props) => {
 class RequestLayer extends React.Component {
   componentWillMount() {
     getData(this.props);
+    window.addEventListener('resize', this.props.updateWindowSize);
```

Comment:
I think you can re-render based on a prop change passed from redux-responsive?  So you don't need to add your own event listener